TapGoods/tapgoods-wp
# update phpcs linting config and first pass fixes

## Ticket
https://tapgoodsdevelopment.atlassian.net/browse/WOR-35

## Purpose
Adds php code sniffer as a project dependence via composer for linting and wordpress code standards 

## Areas to test
Test that php linting is working with wordpress standards
-  On a fresh install of the project after running `docker compose up -d` open the project in VS Code
- Search and Install ``PHP Sniffer & Beautifier Docker`` extension by mtbdata
- Create a new php file and add the following code snippet
```
<?php

function ($foo) 
{

}
```
You should see multiple syntax highlights and be able to view the problems with CTRL(CMD)+Shift+M
![image](https://github.com/TapGoods/tapgoods-wp/assets/156693453/c39057f1-13fb-4758-9043-30f64c5c83d3)


## Changes
- Adds a composer container service the docker compose file that will be responsible for running composer commands
- Adds a composer.json file with the phpcs and wordpress standards dependencies
- Adds .vscode>settings.json file to configure the phpcs docker resolution settings
- Adds phpcs.xml file with wordpress standards directives
